awx_collection/plugins/modules/tower_inventory_source_update.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/python
+# coding: utf-8 -*-
+
+# (c) 2020, Bianca Henderson <bianca@redhat.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: tower_inventory_source_update
+author: "Bianca Henderson (@beeankha)"
+short_description: Update inventory source(s).
+description:
+    - Update Ansible Tower inventory source(s). See
+      U(https://www.ansible.com/tower) for an overview.
+options:
+    inventory:
+      description:
+        - Name of the inventory that contains the inventory source(s) to update.
+      required: True
+      type: str
```

Comment:
Here, again, I think it would make the most sense to add an `organization` field to help with the lookup of the inventory so that you don't get into an unresolvable "duplicates returned" situation.